pg_bm25/test/sql/search_config.sql

```diff
@@ -3,11 +3,9 @@ SELECT id, description, rating, category FROM search_config WHERE search_config
 -- With trailing delimiter
 SELECT id, description, rating, category FROM search_config WHERE search_config @@@ 'category:electronics:::';
 -- With limit
-SELECT id, description, rating, category FROM search_config WHERE search_config @@@ 'category:electronics:::limit=2';
--- With limit and trailing &
-SELECT id, description, rating, category FROM search_config WHERE search_config @@@ 'category:electronics:::limit=2&';
+SELECT id, description, rating, category FROM search_config WHERE search_config @@@ 'category:electronics' LIMIT 2;
 -- With limit and offset
-SELECT id, description, rating, category FROM search_config WHERE search_config @@@ 'category:electronics:::limit=2&offset=1';
+SELECT id, description, rating, category FROM search_config WHERE search_config @@@ 'category:electronics:::offset=1' LIMIT 2;
```

Comment:
Is there any reason for a user to still put `limit=` in the config string? Should probably remove `limit` from `SearchQueryConfig` in `index_access/utils.rs` if we're not going to use it anymore.

Reply:
It's been removed! The diff you're looking at is the old code.